_providers/outlook.com.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Outlook.com
+website: https://outlook.live.com/mail
+domains:
+ - hotmail.com
+ - outlook.com
+ - office365.com
+ - outlook.com.tr
+credentials: 
+ - emailPass
+ - emailAppPass
+ - Oauth
+ status:
+ state: PREP
+ date: 2019-09
+### Login Instructions
```

Comment:
before this line `---` is missing to close the yaml font matter.

Reply:
Also this heading should be called `## Preparations` or atleast `## Preparations` should be before `### Login Instructions`